mods/ra2/rules/allied-structures.yaml

```diff
@@ -349,6 +349,20 @@ gaairc:
 	SpawnSurvivors:
 		DeathTypes: ExplosionDeath, BulletDeath
 		Actors: e1, e1
+	SpawnNeighboringActors:
+		ActorTypes: gapad
+		Locations: 1,0, 1,1, 2,1
+
+gapad:
```

Comment:
Why, technically, are we splitting this into multiple actor types?
Is this required to work around an engine issue?
If not, to me this seems like (prematurely) building more than we need.

Reply:
There can't be more than one docking activity per actor. This is an engine limitation which can't be resolved here.